subsys/bluetooth/host/Kconfig

```diff
@@ -421,12 +421,16 @@ if BT_GAP_PERIPHERAL_PREF_PARAMS
 config BT_PERIPHERAL_PREF_MIN_INT
 	int "Peripheral preferred minimum connection interval in 1.25ms units"
 	default 24
-	range 6 3200
+	range 6 65535
```

Comment:
Maybe add another config option, "SPECIFY PREF_MIN"? That way you can specify the value and have range validation, or it will be set to 0xffff?

Reply:
Not sure I follow, or do you mean add a bool config option which enables specifying the value? It is possible to do something like that but Im not sure it is worth it since we would have to add another 3 config option to achieve something similar.

Reply:
Yes, that is what I meant. It might be a lot of options, but I think it makes it better.